src/databricks/sql/client.py

```diff
@@ -17,7 +17,7 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_RESULT_BUFFER_SIZE_BYTES = 10485760
+DEFAULT_RESULT_BUFFER_SIZE_BYTES = 104857600
```

Comment:
10485760 bytes was too small for cloud fetch to work as one result link may represent more than that many bytes.

Reply:
Is this number arbitrary? Should we make it configurable?

Reply:
The increase to 104857600 is somewhat arbitrary. I'll provide a follow-up PR to make this configurable

Reply:
The increase to 104857600 is somewhat arbitrary - x10 from the original simply has worked for cloud fetch. I'll provide a follow-up PR to make this configurable